cmd/ttn-lw-cli/commands/clients.go

```diff
@@ -68,6 +70,10 @@ var (
 		Short:   "List clients",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			paths := util.SelectFieldMask(cmd.Flags(), selectClientFlags)
+			dropped := util.DropForbiddenFieldMaskPaths(paths, forbiddenSelectClientFlags)
+			for _, path := range dropped {
+				logger.Warnf("Ignoring forbidden field mask `%s`", path)
+			}
```

Comment:
So what happens with `dropped`?

Reply:
The `dropped` paths had already been dropped from the `paths`.

No longer relevant, switched to using `ttnpb.ExcludeFields` instead.